<!DOCTYPE HTML>

<html>
	<head>
		<title>About. Asma Nouira</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Header -->
			<div id="header">

				<div class="top">

					<!-- Logo -->
						<div id="logo">
						
							<h1 id="title">Asma Nouira</h1>
							<p>PhD Student </p>
							<p>asma.nouira@mines-paristech.fr </p>
						</div>

					<!-- Nav -->
						<nav id="nav">
							<ul>
								<li><a href="#top" id="top-link"><span class="icon solid fa-home">Intro</span></a></li>
								<li><a href="#about" id="about-link"><span class="icon solid fa-user">About me</span></a></li>
								<li><a href="#conf" id="conf-link"><span class="icon solid fa-comments"> Talks and Posters </span></a></li>
								<li><a href="#pub" id="pub-link"><span class="icon solid fa-edit">Publications</span></a></li>
								<li><a href="#git" id="git-link"><span class="icon solid fa-keyboard">GitHub Repositories</span></a></li>
								<li><a href="#teach" id="teach-link"><span class="icon solid fa-chalkboard-teacher">Teaching</span></a></li>
							</ul>
						</nav>

				</div>

				<div class="bottom">

					<!-- Social Icons -->
						<ul class="icons">

							<li><a href="https://github.com/asmanouira" class="icon brands fa-github"><span class="label">Github</span></a></li>
							<li><a href="mailto:<nowiki>asma.nouira@mines-paristech.fr" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
							<li><a href="https://twitter.com/AsmaaNouira" class="icon brands fa fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="https://www.linkedin.com/in/asma-nouira" class="icon brands fa fa-linkedin"><span class="label">Linkedin</span></a></li>
						</ul>

				</div>

			</div>

		<!-- Main -->
			<div id="main">

				<!-- Intro -->
					<section id="top" class="one dark cover">
						<!-- <div style="background-image: url('assets/css/images/ie/background.jpeg');">-->
						<div class="container"> 

							<header>
								
								<h3 class="alt"> 
								<p align="justify">
								I will start a postdoc position in March 2023 at <a href="http://www.mines-paristech.fr/"> Université Paris Cité </a> under the supervision of Marie Verbanck.
								I will mainly work on developing machine learning methods to leverage pleiotropy in human genetic architecture to build a map of pleiotropy.
							        From January 2019 to July 2022, I have been a <strong>PhD Student</strong> in Computational Biology
							    at  <a href="http://cbio.mines-paristech.fr/"> CBIO team </a> in <a href="http://www.mines-paristech.fr/"> MINES ParisTech </a> and <a href="https://institut-curie.org/"> Institut Curie </a>. I am working under the supervision of <a href="http://cazencott.info/"> Chloé-Agathe Azencott. </a></h3> </p>
								<p align="justify">
								I have defended my PhD in 13th July 2022, my thesis manuscript is online <a href="https://pastel.archives-ouvertes.fr/tel-03850681"> here.
								The main goal of my thesis is to provide a stable framework for biomarker discovery in multi-locus Genome-Wide Association Studies (GWAS) using Machine Learning, essentially feature selection models to deal with high-dimensional data. The aim is to find association between the genotype and the phenotype, we are conducting our analysis on cancer phenotype. 
								I have adressed many challenges that lay ahead an association analysis such as: genetic population stratification, linkage disequilibrium patterns clustering and the stability of the selection ...
							</header>

							<footer>
								<a href="#about" class="button scrolly">About me </a>
							</footer>
						</div>
					</section>

				<!-- Portfolio -->
					<section id="about" >
						<div class="container">

							<header>
								<h2>About me</h2>
							</header>

							<p align="justify">In 2017, I obtained my engineer diploma on embedded systems in National Engineering School of Sousse (ENISo), Tunisia. After that, I got my master degree in Intelligent Communicating Systems in 2018. I did an internship at East Paris Institute of Chemistry and Materials Science (ICMPE) in Paris, I was working on Machine learning techniques applied on chemistry to discover novel chemical compounds for hydrogen storage. This project was supervised by <a href= "http://www.icmpe.cnrs.fr/spip.php?article627"> Jean-Claude Crivello </a> and <a href="https://integromics.fr/~nsokolovska/"> Nataliya Sokolovoska </a>. Since January 2019, I started my PhD project at CBIO, a team directed by <a href="http://members.cbio.mines-paristech.fr/~twalter/"> Thomas Walter </a>. I am mostly interested on Machine Learning and Deep Learning applications. At CBIO, I am looking forward to resolving biological problems in order to find association between the human genome and cancer using Machine Learning techniques.</p>

							
						</div>
					</section>

				<!-- About Me -->
					<section id="conf" >
						<div class="container">

							<header>
								<h2>Talks and Posters</h2>
							</header>


							<ul style="list-style-type:disc; text-align: left;">
							<li> <strong>March 17, 2022.</strong> Multi-task group Lasso for Genome Wide Association Studies. The Junior Seminar in Artificial Intelligence and Digital Healthcare, Paris (France). [Invited talk]
							<li> <strong>February 3, 2022.</strong> Multi-task group Lasso for Genome Wide Association Studies in diverse populations. Institut Curie, Paris (France). <a href="files/Curie2022.pdf"> [Talk] </a><br /> </li>
							<li> <strong>January 6, 2022.</strong> Multi-task group Lasso for Genome Wide Association Studies in diverse populations. Pacific Symposium on Biocomputing (PSB 2022), Hawaii (USA). <a href="files/PSB2022.pdf"> [Talk] </a><br /> </li>
							<li> <strong>October 25, 2021.</strong> Multi-task group Lasso for Genome Wide Association Studies in diverse populations. NutriOmics siminar. <a href="files/NutriOmics_seminar.pdf"> [Invited talk] </a><br /> </li>
							<li> <strong>July 29, 2021.</strong> Multi-task group Lasso for admixed populations in Genome Wide Association Studies. Intelligent Systems for Molecular Biology (ISMB 2021), MLCSB COSI.  <a href="files/ISMB_2021.pdf"> </a>[Talk] <a href="files/146AsmaNouiraMLCSB_Poster.pdf">[Poster]</a><br /> </li>
							<li> <strong>January 28, 2021.</strong> Stable Multi-task feature selection approach for Genome Wide Association Studies. Institut Curie, Paris (France). <a href="files/Curie2021.pdf">[Talk]</a><br /> </li>
							<li> <strong>March 11, 2020.</strong> Multi-task group Lasso correcting for population stratification in Genome Wide Association Studies. Intitut Curie, Paris (France).<a href="files/Lab_meeting11.03.20_AsmaNOUIRA.pdf">[Talk]</a><br /> </li>
							<li> <strong>January 23, 2020.</strong> Multi-task group lasso for Genome Wide Association Studies. Statistical Methods for Post Genomic Data (SMPGD 2020), Institut Pasteur, Paris (France).  <a href="files/smpgd.pdf">[Poster]</a></li>
							

							</p>
							</ul>
						</div>
					</section>

				<!-- Contact -->
					<section id="pub" >
						<div class="container">

							<header>
								<h2>Publications</h2>
							</header>
							<ul style="list-style-type:disc; text-align: left;">
		 					<li> <strong>Nouira. A.</strong>, Azencottt. C-A, Multi-task group lasso for Genome Wide Association Studies in diverse populations. Pacific Symposium on Biocomputing (PSB) 2022. <a href ="https://www.biorxiv.org/content/10.1101/2021.08.02.454499v2.full.pdf">[Full paper]</a> <a href="files/Supplementary_materials_MuGLasso.pdf">[Supplementary Materials]</a></li>
							</p>
							</ul>
							<ul style="list-style-type:disc; text-align: left;">
							<li> <strong>Nouira. A </strong>, Sokolovska. N, Crivello. J-C, <a href ="http://ceur-ws.org/Vol-2350/paper18.pdf"> CrystalGAN: Learning to Discover Crystallographic Structures with Generative Adversarial Networks.</a>  AAAI Spring Symposium: Combining Machine Learning with Knowledge Engineering 2019.</li>
							</p>
							</ul>
		                                        

						</div>
					</section>


					<section id="git" >
						<div class="container">

							<header>
								<h2>GitHub Repositories</h2>
							</header>

							<ul style="list-style-type:disc; text-align: left;">
							<li> <a href = "https://github.com/asmanouira/MuGLasso_GWAS"> MuGLasso_GWAS </a>: Multitask group lasso for Genome Wide Association Studies in diverse populations. </li>
							<li> <a href= "https://github.com/asmanouira/GWAS-admixed-population-simulator"> GWAS-admixed-population-simulator </a>: Simulating case-control admixed-population GWAS data in PLINK format. </li>
							<li> <a href= "https://github.com/asmanouira/CrystalGAN"> CrystalGAN </a>: Learning to Discover Crystallographic Structures with Generative Adversarial Networks. </li>
							<li> <a href = "https://github.com/asmanouira/Crystal-tools"> Crystal-tools </a>: Useful tools for the preprocessing of CrystalGAN. </li>	
							
							</ul>
		

						</div>
					</section>
					
					<section id="teach" >
						<div class="container">

							<header>
								<h2>Teaching</h2>
							</header>
							
							<ul style="list-style-type:disc; text-align: justify;">
							<li>  <strong>November, 2021</strong>: Teaching assistant. Introduction to Machine Learning: <a href ="https://github.com/asmanouira/TP_SVM_2019/blob/master/TP_SVM_13.11.2019.ipynb"> Support Vector Machines  </a>  at MINES ParisTech. </li> 
							<li>  <strong>March, 2021 </strong>: Teaching assistant. Workshops of <a href="http://cazencott.info/index.php/pages/LSML-21-Large-Scale-Machine-Learning">Large-Scale Machine Learning and Data Mining 2021 </a> at MINES ParisTech. </li>
							<li>  <strong>November, 2019</strong>: Teaching assistant. Introduction to Machine Learning:  <a href ="https://github.com/asmanouira/TP_SVM_2019/blob/master/TP_SVM_13.11.2019.ipynb"> Support Vector Machines  </a>  at MINES ParisTech. </li> 
							<li>  <strong>March, 2019 </strong>: Teaching assistant. Workshops of <a href="http://cazencott.info/index.php/pages/LSML-19-Large-Scale-Machine-Learning">Large-Scale Machine Learning and Data Mining 2019 </a> at MINES ParisTech. </li>
							
							</p> 
							</ul>




		

						</div>
					</section>
			</div>

		<!-- Footer -->
			<div id="footer">



		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
